ci/test-frozen-abi.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# Easily run the ABI tests for the entire repo or a subset
+#
+
+set -e
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+
+# shellcheck source=ci/rust-version.sh
+source "$here/rust-version.sh" nightly
+
+packages=$(cargo +"$rust_nightly" metadata --no-deps --format-version=1 | jq -r '.packages[] | select(.features | has("frozen-abi")) | .name')
+for package in $packages; do
+  echo "--- cargo +$rust_nightly test -p $package --features frozen-abi --lib -- test_abi_ --nocapture"
```

Comment:
```suggestion
  cargo_cmd="cargo +$rust_nightly test -p $package --features frozen-abi --lib -- test_abi_ --nocapture"
  echo "--- $cargo_cmd"
```
And then also `$cargo_cmd` in the line below